Pipfile

```diff
@@ -0,0 +1,21 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+future = "*"
+futures = "==3.1.1"
+requests = "*"
+simplejson = "*"
+tornado-json = "*"
+tornado = "*"
+cloudpickle = "*"
+decorator = "*"
+python-dateutil = "*"
+genson = "*"
+jsonschema = "*"
+"809dc83" = {path = "./tabpy-client", edtiable = true}
+numpy = "*"
+
+[dev-packages]
```

Comment:
One potential weakness here is that Pipenv doesn't support specifying multiple Python versions:

https://github.com/pypa/pipenv/issues/1050

If someone is trying to develop on a non-supported Python version this wouldn't stop them from doing so, but as far as distributions are concerned you'd be guarding against that in the `setup.py` requirements